src/main/java/picard/sam/SetNmMDAndUqTags.java

```diff
@@ -48,19 +47,19 @@
  * @author Yossi Farjoun
  */
 @CommandLineProgramProperties(
-        usage = SetNmAndUqTags.USAGE_SUMMARY + SetNmAndUqTags.USAGE_DETAILS,
-        usageShort = SetNmAndUqTags.USAGE_SUMMARY,
+        usage = SetNmMdAndUqTags.USAGE_SUMMARY + SetNmMdAndUqTags.USAGE_DETAILS,
+        usageShort = SetNmMdAndUqTags.USAGE_SUMMARY,
         programGroup = SamOrBam.class
 )
-public class SetNmAndUqTags extends CommandLineProgram {
-    static final String USAGE_SUMMARY = "Fixes the UQ and NM tags in a SAM file.  ";
-    static final String USAGE_DETAILS = "This tool takes in a SAM or BAM file (sorted by coordinate) and calculates the NM and UQ tags by comparing with the reference."+
+public class SetNmMdAndUqTags extends CommandLineProgram {
```

Comment:
This is a breaking change...can we have a deprecated version of the old class-name? for a few releases?